index.html

```diff
@@ -192,7 +192,7 @@
         <div class="col-md-4 col-md-offset-0 col-lg-3 col-lg-offset-1">
           <h3>Where?</h3>
           <p>
-            <img src="https://www.startupedmonton.com/images/logo.png"  class="colOpenGraphic" />
+            <img src="startup-edmonton.png"  class="colOpenGraphic" />
```

Comment:
This change was included in this branch as I had to list the files the cache in the service worker and realized this was missing.